pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -96014,7 +96014,6 @@ self: {
        description = "The core of an IDE";
        license = stdenv.lib.licenses.asl20;
        hydraPlatforms = stdenv.lib.platforms.none;
-       broken = true;
```

Comment:
You'll have to remove the `hydraPlatforms = stdenv.lib.platforms.none;` lines here as well.

If you don't remove them, then Hydra won't build these packages on any platforms, and you won't get builds from the cache.

Reply:
Oh, that makes total sense.